docs/test.rst

```diff
@@ -32,4 +32,14 @@ Because django will not create tenants for you during your tests, we have packed
             response = self.c.get(reverse('user_profile'))
             self.assertEqual(response.status_code, 200)
 
+
+Running tests faster
+--------------------
+Using the ``TenantTestCase`` can make running your tests really slow quite early in your project. This is due to the fact that it drops, recreates the test schema and runs migrations for every ``TenantTestCase`` you have. If you want to gain speed, there's a ``FastTenantTestCase`` where the test schema will be created and migrations ran only one time. The gain in speed is noticiable but be aware that by using this you will be perpertraiting state between your test cases, please make sure your they wont be affected by this.
```

Comment:
@bernardopires this looks better, will update it. One small change, I believe it's more precise to say migrations will be created and ran once per time you run the test suite. The `TenantTestCase` runs migrations once per `TestCase` class.